src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/WorkloadManifest.json.in

```diff
@@ -119,7 +119,7 @@
       "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
```

Comment:
Probably unrelated to this, but are we missing arm64 from the platforms here? (It just kind of caught my eye)

Reply:
https://github.com/dotnet/runtime/pull/83162

Reply:
oh wait... good question
@steveisok ?

Reply:
Yes, that's another one that should be there.

Reply:
https://github.com/dotnet/runtime/pull/83237